pkg/controllers/managementuser/rbac/prtb_handler.go

```diff
@@ -106,6 +106,11 @@ func (p *prtbLifecycle) syncPRTB(binding *v3.ProjectRoleTemplateBinding) error {
 	}
 
 	if err := p.m.ensureRoles(roles); err != nil {
+		if errors.Is(err, errNotReady) {
```

Comment:
Then we would have two different implementations in the same file, it's more important to be consistent imo

Reply:
I understand your point, and we may just disagree, which is okay. In this instance, since the std library has a one-for-one replacement, I rather change the entire file if we care about consistency over adding new code with the deprecated package.  I am not willing to die on this hill so we can resolve and move on.

Reply:
also, this package has larger problems than using the `pkg/errors`

Reply:
I added a commit to change it for the entire module, now no one else will have to have this debate :)